commonslibrary/settings/base.py

```diff
@@ -237,3 +240,18 @@
 
 # Improved cookie security
 CSRF_COOKIE_HTTPONLY = True
+
+AUTH_USER_MODEL = 'accounts.User'
+
+# yapf: disable
+CKEDITOR_CONFIGS = {
+    'default': {
+        'skin': 'moono-lisa',
+        'toolbar_Basic': [['Source', '-', 'Bold', 'Italic']],
+        'toolbar': [{
+            'name': 'basicstyles',
+            'items': ['Bold', 'Italic', 'Underline', 'Strike', 'Image', 'Link'],
+        }],
+    }
+}
+# yapf: enable
```

Comment:
Could you also:

- Set sorl logger to console (to prevent it spamming sentry)
- Upgrade Pillow
- Add our standard thumbnail settings

https://github.com/developersociety/helprefugees/pull/30/files is a typical example.

Reply:
Are all these standards should be in the Django template? Otherwise, don't think anyone will be remembering to add these things in the upcoming projects.

Reply:
It was part of the glitter 1.8 template - we'll probably need something separate for the 1.11 and non-glitter/wagtail projects.